prover/zkevm/arithmetization/definition.go

```diff
@@ -87,7 +87,7 @@ func (s *schemaScanner) scanConstraints() {
 	corsetCSs := s.Schema.Constraints().Collect()
 
 	for _, corsetCS := range corsetCSs {
-		name := corsetCS.String()
+		name := fmt.Sprintf("%v", corsetCS)
```

Comment:
@AlexandreBelling 
There were actually compile errors with the latest go-corset.  
```
zkevm/arithmetization/definition.go:90:20: corsetCS.String undefined (type "github.com/consensys/go-corset/pkg/schema".Constraint has no field or method String)
zkevm/arithmetization/definition.go:234:57: e.String undefined (type air.Expr has no field or method String)
```